data/human/free worlds 0 prologue.txt

```diff
@@ -1748,24 +1972,35 @@ mission "FW Bounty 3"
 			`Once again, as you are walking through the spaceport a Free Worlds officer flags you down. "We've got another bounty hunting mission for you," he says. "This one is a bit more dangerous; this pirate ship has weapons we've never seen before, that may be alien in origin. Interested in helping us to destroy it?"`
 			choice
 				`	"Yes, I'm sure my ship will be up to the challenge."`
+					goto accept
+				`	"Alien in origin? What sort of weapons does it have?"`
 				`	"Sorry, I'm not interested in giving the Free Worlds more assistance."`
 					decline
-			`	"Great," he says. "Your target is named the Silverhawk. After we nearly destroyed it, we think it went to the Shaula system to hide out. Which means that you'll probably have to deal with other pirates at the same time. The Silverhawk has weapons that seem to interfere with a ship's electrical systems, draining power from them. Good luck, Captain."`
+			
+			`	"It has a pair of strange cannons that seem to interfere with a ship's electrical systems, draining its power and causing its weapons to malfunction. It's similar to those lasers the Quarg have, which is why we think the weapons are alien technology."`
```

Comment:
Do human ships actually last long enough under fire from Quarg ships for the ionisation to be apparent?
And, honestly, telling me they have weapons like the Quarg's isn't going to encourage me to take this job.

Reply:
> telling me they have weapons like the Quarg's isn't going to encourage me to take this job.

Well I did bump the pay up.